src/types/globals.d.ts

```diff
@@ -0,0 +1,8 @@
+declare type EmptyObject = Record<string, never>
+
+declare interface Bridge {
+  send: (channel: string, data?: any) => void,
+  receive: (channel: string, func: any) => void
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/ferrwan/brankas/pullRequest?prid=7906180)
```suggestion
  receive: (channel: string, func: unknown) => void
```